app/services/books/book_rating.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Books
+  class BookRating < ::Callable
+    def initialize(book:)
+      @book = book
+    end
+
+    def call
+      votes = {}
+      table = Rating
+              .select('ratings.title, count(ratings)')
```

Comment:
if you remove this line and add `count` at the end of this query, you will receive the same result, but without your votes and table stuff (17 and 18 lines)